src/Components/Footer.js

```diff
@@ -0,0 +1,19 @@
+import react from "react";
+
+const Footer = (props) =>{
+            console.log('props', props);
+
+    return(
+        <footer>
+            <ul>
+                {props.address.map((element, index) => {
+                    console.log(element)
+                    console.log(index)
+                    return <li key={element}>{element}</li>
+
+                })}
+            </ul>
+        </footer>
+    )
+}
+export default Footer
```

Comment:
When a file should only ever export a single element as default you can inline it to the declaration, e.g.

```
export default function Footer(props) {
  ...
}
```